src/status_im/extensions/capacities/components.cljs

```diff
@@ -51,7 +52,7 @@
                                         (when style {:style style}))] children))
 
 (defn image [{:keys [source uri style]}]
-  [react/image (merge {:style (merge {:width 100 :height 100} style)} {:source (if source source {:uri uri})})])
+  [rn-dependencies/fast-image (merge {:style (merge {:width 100 :height 100} style)} {:source (if source source {:uri uri})})])
```

Comment:
we need to replace usage of class inside react/image itself, here https://github.com/status-im/status-react/blob/c2b55233225f6e4d59c9ca5174b5261cc1137fec/components/src/status_im/ui/components/react.cljs#L78, thanks